config/eslintrc_react.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
     // https://github.com/yannickcr/eslint-plugin-react
     'rules': {
         'react/boolean-prop-naming': 0, // We don't want to give special weight to `boolean`.
+        'react/button-has-type': 1,
```

Comment:
[By the spec](https://html.spec.whatwg.org/multipage/form-elements.html#attr-button-type), the default value for`type` attribute of `button` element is `type="submit"`.

If the default value were `type="button"`, I think it would be better that we don't restrict about this (but it's not).

I agree this rule marked as warning.